### PR TITLE
Upgrade 0.27

### LINKED ISSUE #20: Upgrade 0.27
Upgrade 0.27 this module

### PR DESCRIPTION
**Pull request by antopalidi**
#### :tophat: What? Why?
Upgrade Decidim 0.27.5

#### :pushpin: Related Issues
- Fixes #20

### CODE REVIEW

**Comment by antopalidi** [comment]
@microstudi This pull request is ready for review

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 Following Semantic Versioning 2.
 
+## Version 0.6.0 (MINOR)
+
```
Comment (microstudi) [inline CHANGELOG.md:5]:
```suggestion
- Adds compatibility with Decidim 0.27
```

File: app/services/decidim/via_oberta/api/request.rb [hunk 2]
```diff
@@ -141,7 +141,7 @@ def request_body
                             <ns2:numExpediente>#{request_id}</ns2:numExpediente>
                             <ns2:tipoDocumentacion>#{document_type_id}</ns2:tipoDocumentacion>
                             <ns2:documentacion>#{document_id}</ns2:documentacion>
-                            <ns2:codigoMunicipio>#{municipal_code}</ns2:codigoMunicipio> 
+                            <ns2:codigoMunicipio>#{municipal_code}</ns2:codigoMunicipio>#{" "}
```
Comment (microstudi) [inline app/services/decidim/via_oberta/api/request.rb:144]: is this a mistake?

**Review by microstudi — CHANGES_REQUESTED** [review]
Good job Anna,

I've left a couple of comments, please check it out.
Also, please update the README file of any reference for version 0.26 to version 0.27!

File: app/services/decidim/via_oberta/api/request.rb (hunk 2 above)
Comment (antopalidi) [reply app/services/decidim/via_oberta/api/request.rb]: yes, there was a space, I removed it

**Review by microstudi — APPROVED** [review]
great job!

**Pull request merged**